pylint/testutils/_primer/primer_run_command.py

```diff
@@ -73,5 +75,23 @@ def _lint_package(self, data: PackageToLint) -> list[Message]:
         arguments = data.pylint_args + enables + disables
         output = StringIO()
         reporter = JSONReporter(output)
-        Run(arguments, reporter=reporter, exit=False)
-        return [JSONReporter.deserialize(m) for m in json.loads(output.getvalue())]
+        print(f"Running 'pylint {', '.join(arguments)}'")
+        pylint_exit_code = -1
+        try:
+            Run(arguments, reporter=reporter)
+        except SystemExit as e:
+            pylint_exit_code = int(e.code)
+        readable_messages: str = output.getvalue()
+        messages: list[OldJsonExport] = json.loads(readable_messages)
+        astroid_errors: list[Message] = []
+        other_fatal_msgs: list[Message] = []
+        if pylint_exit_code % 2 == 0:
+            print(f"Successfully primed {package_name}.")
+        else:
+            print(
```

Comment:
Shouldn't this be the other way around? First determine where the fatal errors are coming from and then printing them with the according header?